docs/ACCEPTANCE_TESTS.md

```diff
@@ -0,0 +1,97 @@
+# Rule acceptance tests 
+
+## Goal
+Execute the validator against all datasets (latest version hosted in the MobilityDatabase) and quantify the effect of a code change on all of them.  
+The source of truth is defined as the latest stable version of the validator (the on available on [the master branch of this repository](https://github.com/MobilityData/gtfs-validator/tree/master)). 
+
+For the latest version of all GTFS datasets from the MobilityArchives, the validation report from both the snapshot and the source of truth are compared. An acceptance test report is generated: it quantifies for each agency/dataset the number of new errors (as defined [here](https://github.com/MobilityData/gtfs-validator/blob/master/RULES.md#definitions)) in introduced.
+   
+## Process description
+
+![steps](https://user-images.githubusercontent.com/35747326/120213646-6fb7bb80-c201-11eb-8520-39c88e9753d0.png)
+
+## Github Actions
+The logic for this process is defined in [`acceptance_test.yml`](../.github/workflows/acceptance_test.yml).
+This workflow:
+1. packages the `comparator` module;
+1. downloads the latest release of the validator;
+1. defines a matrix of urls (fetched from the Mobility archives) that will be used in further validation process; 
+
+On each of these urls:
+1. the latest stable version of the validator is executed and the validation report is output as JSON (under `reference.json`);
```

Comment:
I would avoid words like "stable" and just repeat the same word throughout to be clear what exactly is being used.